torchvision/prototype/constants.py

```diff
@@ -0,0 +1,34 @@
+# Maybe a good place to put all enums that are relevant to more than module?
```

Comment:
We need these enums in features and transforms. Thus, we can't put them in these modules if we want to avoid cyclic dependencies. They could go into kernels, but IMO they are misplaced there. This is why I created a separate module. We can also expose them under `torchvision.prototype` (or `torchvision`) later to make it more convenient for users.